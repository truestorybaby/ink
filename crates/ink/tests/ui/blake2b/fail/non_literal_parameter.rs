const INPUT: &str = "test";
const _: [u8; 32] = ink::blake2x256!(INPUT);

fn main() {}
