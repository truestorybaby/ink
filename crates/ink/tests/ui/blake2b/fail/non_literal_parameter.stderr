error: expected string or byte string literal as input: expected literal
 --> tests/ui/blake2b/fail/non_literal_parameter.rs:2:38
  |
2 | const _: [u8; 32] = ink::blake2x256!(INPUT);
  |                                      ^^^^^
